manual/development.md

```diff
@@ -16,3 +16,189 @@ Do 3 steps:
   2. Add an entry into config/enabled.yml or config/disabled.yml
   3. Implement your new cop in the generated file!
 ```
+
+## Implementing the cop
+
+You can start by learning the
```

Comment:
I think people should start by learning a bit about `parser`. :-)

Reply:
That is true! I'll improve this part to make it easy to comprehend. Node pattern is the second step I think.